### DISCUSSION TITLE
En busca de equipo

### DISCUSSION THREAD

**Discussion by CELFIE14** [Launch-X-Latam/MisionFrontEnd#506] [Team Posts]
Hola a todoooos también busco equipo, estoy aprendiendo y nos podemos ayudar muchísimo, soy de México y tengo la carrera de Contabilidad y además estoy ya en 7mo semestre de Ing. en Desarrollo de Software, se algo de diseño gráfico y tengo muchas ganas de trabajar y seguir aprendiendo!!!! Aquí dejo mi correo estoy disponible para conformar equipo.
lx02482@innovacción.mx
Gracias por leer !!!!

**Comment by Alisvaz** [comment]
Hola Celfie, yo también estoy en búsqueda de equipo. 
Puedo entrar a tu equipo? 
lx04406@innovaccion.mx

**Comment by NickArregoces** [comment]
hola puedo entrar a su grupo de hackaton, mi correo es: [lx14256@innovaccion.mx] me llamo Nicolas soy de Colombia

**Comment by HoracioTorrez** [comment]
Hola si aun tienen espacio y estan en busca de un viejo de 39 años nuevo en este mundo del desarrollo me encantaria estar en su equipo lx15713@innovaccion.mx también soy de méxico. Gracias.

**Comment by Salcedotoro** [comment]
Hola. Mi nombre es José Salcedo. Soy de Venezuela. También estoy buscando
con quién formar equipo. Este es mi correo: ***@***.***

**Comment by Carlopsd** [comment]
Hola soy Carlos, si aun buscas para tu equipo, mi correo de innovación es lx16072@innovaccion.mx 
Manejo lenguales de python, html, css/scss, javascript y también manejo git.

**Comment by Sacoma1** [comment]
Hola soy miguel, igual soy de mexico y estoy en busca de equipo, este es mi correo lx16970@innovaccion.mx

**Comment by Rosruiz** [comment]
Hola, estoy en búsqueda de un equipo,  me llamo Rosa, me puedo unir?   
Este es mi correo  lx02117@innovaccion.mx

**Comment by LuisGGDev** [comment]
Buen dia, a mi me interesa entrar al equipo, soy Luis, ingeniero en sistemas computacionales, y me encunetro con ganas de pertenecer a un equipo, soy de Mexico.
este es mi correo: lx16528@innovaccion.mx

**Comment by stgosm** [comment]
@CELFIE14 Tenemos un equipo y nos hace falta un integrante para completarlo, te gustaría que te involucráramos?

**Comment by IngCarrillo12** [comment]
Todavía hay cupo? lx08898@innovaccion.mx

**Comment by Salcedotoro** [comment]
Si no lo han completado incluyanme. Ix09513

**Comment by FrankAV27** [comment]
Hola mi nombre es Franklin y quisiera unirme a tu grupo mi correo es: lx09414@innovaccion.mx

**Comment by carchirinos** [comment]
Hola, si se forma el grupo estoy interesadoe en unirme, les dejo mi correo  lx15343@innovaccion.mx creo que los equipos son de hasta 5 personas. No se si ya alguien creo el grupo en teams??
